addon/internal-session.js

```diff
@@ -111,8 +111,8 @@ export default ObjectProxy.extend(Evented, {
     set(this.content, 'authenticated', authenticatedContent);
     this._bindToAuthenticatorEvents();
 
-    return this._updateStore().then(() => {
-      this.endPropertyChanges();
```

Comment:
I think the second call is in the rejection though. If `_updateStore` returns a resolving promise I think `endPropertyChanges` wouldn't be called at all anymore?

Reply:
@BryanCrotaz: any update on this?